apps/api/src/domain/models/context/edge-connection.entity.ts

```diff
@@ -183,7 +183,9 @@ export class EdgeConnection extends Aggregate {
         if (connections.some(idEquals(this.id)))
             allErrors.push(new AggregateIdAlreadyInUseError(this.getCompositeIdentifier()));
 
-        allErrors.push(...this.validateMembersState(externalState));
+        const inconsistentStateErrorsFromMemebers = this.validateMembersState(externalState);
```

Comment:
(sp) Memembers

Reply:
Fixed. Now the only occurrence of `meme`  in our code base is in getWelco**meMe**message.